gradle.properties

```diff
@@ -0,0 +1,64 @@
+group=io.ballerina.stdlib
+version=1.0.0-SNAPSHOT
+
+puppycrawlCheckstyleVersion=8.18
+checkstyleToolVersion=7.8.2
+githubSpotbugsVersion=4.0.5
+githubJohnrengelmanShadowVersion=5.2.0
+underCouchDownloadVersion=4.0.4
+researchgateReleaseVersion=2.8.0
+testngVersion=7.4.0
+gsonVersion=2.10
+ballerinaGradlePluginVersion=1.0.0
+
+ballerinaLangVersion=2201.6.0-20230525-070000-e36fc7ac
+
+# Direct Dependencies
+
+# Level 01
+stdlibIoVersion=1.4.0
+stdlibTimeVersion=2.2.4
+stdlibUrlVersion=2.2.3
+
+# Level 02
+stdlibLogVersion=2.7.1-20230426-134900-2076e24
+stdlibOsVersion=1.6.0
+
+# Level 03
+stdlibFileVersion=1.7.1-20230426-145900-748058d
+
+# Level 05
+stdlibHttpVersion=2.7.1-20230427-161400-631151b
+
+# Level 08
+stdlibSqlVersion=1.8.1-20230517-130300-fd0b0e5
```

Comment:
```suggestion
```

Reply:
fixed in [caecb93](https://github.com/ballerina-platform/module-ballerinax-persist.googlesheets/pull/1/commits/caecb93da85c1d1626d09ea912843b8249d12237)

Reply:
[46d7892](https://github.com/ballerina-platform/module-ballerinax-persist.googlesheets/pull/1/commits/46d78928cd81d1b1637013df027a48ea2bc64f67)